index.rst

```diff
@@ -1,171 +1,782 @@
+        <div class="getting-started-card">
+            <h3>From Home Assistant</h3>
+            <p>The easiest way to get started with ESPHome is through the Home Assistant add-on.</p>
+            <ol>
+                <li>Open Home Assistant</li>
+                <li>Go to Settings → Add-ons → Add-on Store</li>
+                <li>Find and install the ESPHome add-on</li>
+                <li>Open the ESPHome dashboard and create your first device</li>
+            </ol>
+            <a href="/guides/getting_started_hassio.html" class="btn btn-primary">Detailed Instructions</a>
+        </div>
+        <div class="getting-started-card">
+            <h3>Command Line</h3>
+            <p>For advanced users who prefer working with the command line.</p>
+            <ol>
+                <li>Install Python 3.11 or higher</li>
+                <li>Install ESPHome: <code>pip install esphome</code></li>
+                <li>Create a configuration file</li>
+                <li>Compile and upload: <code>esphome run my_config.yaml</code></li>
+            </ol>
+            <a href="/guides/getting_started_command_line.html" class="btn btn-primary">Detailed Instructions</a>
+        </div>
+        <div class="getting-started-card">
+            <h3>Ready-Made Projects</h3>
+            <p>Start with a pre-configured project for common use cases.</p>
+            <ol>
+                <li>Browse the project library</li>
+                <li>Select a project that matches your hardware</li>
+                <li>Follow the installation instructions</li>
+                <li>Customize as needed</li>
+            </ol>
+            <a href="/projects/" class="btn btn-primary">Browse Projects</a>
+        </div>
+    </div>
 
-.. _documentation:
+Basic Configuration Example
+---------------------------
 
-Documentation
--------------
+Here's a simple example of an ESPHome configuration file:
 
-You will find all :doc:`ESPHome documentation here<components/index>`.
+.. code-block:: yaml
 
-.. _contributing:
+    # Basic configuration for an ESP32 device
+    esphome:
+      name: living_room_sensor
+      friendly_name: Living Room Sensor
 
-Contributing
-------------
+    # Hardware configuration
+    esp32:
+      board: esp32dev
+      framework:
+        type: arduino
 
-ESPHome depends on and welcomes contributions from our community. If you'd like to contribute, please see
-:doc:`Contributing <guides/contributing>`.
+    # Enable Home Assistant API
+    api:
+      encryption:
+        key: "YOUR_ENCRYPTION_KEY"
+
+    # Enable over-the-air updates
+    ota:
+      password: "YOUR_OTA_PASSWORD"
+
+    # Connect to WiFi
+    wifi:
+      ssid: !secret wifi_ssid
+      password: !secret wifi_password
+      
+      # Enable fallback hotspot if WiFi connection fails
+      ap:
+        ssid: "Living Room Sensor Fallback"
+        password: "YOUR_FALLBACK_PASSWORD"
+
+    # Enable web server
+    web_server:
+      port: 80
```

Comment:
Continuing with simplification -- `port` is optional, I think we could eliminate it here.
```suggestion
```